frontend/pages/Tags.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import ParisMap from "../components/ParisMap";
+import LoadingPage from "./LoadingPage";
+import * as PropTypes from "prop-types";
+import PhotoViewer from "../components/PhotoViewer";
+
+class Tags extends PhotoViewer {
+    constructor(props) {
+        super(props);
+        this.state = {
+            loading: true,
+            tagData: null
+        };
+    }
+
+    async componentDidMount() {
+        try {
+            const response = await fetch(`/api/tag/${this.props.tagName}/`);
```

Comment:
This shouldn't be a separate API call. We can get this information on the backend when the view is called, since we're passing in the tag we're searching for anyway. The view should return necessary information (photos that include tag item)